packages/atomic/src/components/result-lists/grid-display-results.tsx

```diff
@@ -29,10 +29,12 @@ export const GridDisplayResults: FunctionalComponent<ResultsProps> = (
           )}
           result={result}
           engine={props.bindings.engine}
+          store={props.bindings.store}
```

Comment:
is the props.bindings.store a type of datastructure or hashmap?

Reply:
It's an instance of https://stenciljs.com/docs/stencil-store#stencil-store